docs/src/main/asciidoc/spring-native-support.adoc

```diff
@@ -0,0 +1,142 @@
+[#spring-native-support]
+== Spring Native Support
+
+=== Spring Native
+
+Spring Native provides support for compiling Spring Boot applications to native executables using the {graalvm}[GraalVM] {graalvm-native-docs}[native-image] compiler. The native images will bring many advantages, such as instant startup, instant peak performance, and reduced memory consumption. Some Spring Cloud Azure features can also benefit from the Spring Native support, the goal is that Spring Cloud Azure applications can be built as native images without any code modification. See the link:https://docs.spring.io/spring-native/docs/{spring-native-version}/reference/htmlsingle/#overview[Spring Native] for more details.
+
+=== Support
+
+Spring Cloud Azure has been validated against GraalVM and Spring Native, and provides the beta version support. You can try it on your projects if they are using those supported dependencies, and {azure-sdk-java-issues}[raise bugs] or {spring-cloud-azure-native-configuration}[contribute pull requests] if something goes wrong on Spring Cloud Azure level. See the link:https://docs.spring.io/spring-native/docs/{spring-native-version}/reference/htmlsingle/#overview[Support] for more details.
+
+==== Spring Native
+
+Spring Cloud Azure `{project-version}` has been tested against Spring Native `{spring-native-version}` and GraalVM `{graalvm-version}`.
+
+==== Spring Cloud Azure Native
+
+NOTE: Spring Native `{spring-native-version}` has been tested against Spring Cloud Azure Native Configuration `{spring-cloud-azure-native-configuration-version}`.
+
+Spring Cloud Azure provides a dependency `spring-cloud-azure-native-configuration` that is an extension of Spring Native configuration for Spring Cloud Azure level, the Spring Native AOT plugin will combine the `spring-native-configuration` and `spring-cloud-azure-native-configuration` to build Spring Cloud Azure application to native executables. The application built by Spring Cloud Azure does not require additional code modification, it only covers to the code at the Spring Cloud Azure library level.
```

Comment:
for Spring Cloud Azure ~~level~~

Reply:
to build ~~Spring Cloud Azure~~ applications to native executables.

Reply:
No extra modifications are needed to the code that uses Spring Cloud Azure libraries apart from adding the dependency.